### ISSUE TITLE
One set of files per feature type

### ISSUE THREAD

**Issue by jyutzler** [OpenGeoLabs/shapefilemustdie#25]
I think you are missing one of the bigger frustrations with Shapefile - not only do you need multiple files to represent a single feature type but you also need one set of files per feature type so you can get an explosion in the number of files depending on your data model.

**Comment by jachym** [comment]
@jyutzler can you put this into pull request please? ;-)

**Comment by jyutzler** [comment]
Yeah okay I'll try to make a new PR for this and #24. Figure a week.
